package.json

```diff
@@ -10,6 +10,7 @@
     "marked": "^0.8.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-redux": "^7.2.0",
```

Comment:
From this point forward it definitely seems like it should be a new PR. Again, do you think it's work the trouble? It could be a single PR with two version updates as well